sdk/cargo-test-bpf/src/main.rs

```diff
@@ -65,6 +69,22 @@ where
 
     let exit_status = child.wait().expect("failed to wait on child");
     if !exit_status.success() {
+        eprintln!("CARGO-TEST-BPF exited on command execution failure.");
```

Comment:
This also seem too yelly and verbose, wrap behind the verbose flag?

Reply:
as this is relevant mostly for the message about the rerun script, I'll output it only when the script is about to be generated too. with lower case.